src/v/cluster/controller_backend.cc

```diff
@@ -1264,19 +1240,15 @@ controller_backend::reconcile_partition_reconfiguration(
       _self, partition, update.get_resulting_replicas(), cmd_revision);
     if (!update_ec) {
         auto leader = partition->get_leader_id();
-        size_t retries = (rs.cur_operation ? rs.cur_operation->retries : 0);
         vlog(
           clusterlog.trace,
           "[{}] update complete, checking if our node can finish it "
           "(leader: {}, retry: {})",
           partition->ntp(),
           leader,
-          retries);
+          rs.cur_operation ? rs.cur_operation->retries : 0);
```

Comment:
No need to log `retry` if it's not used (or is that some kind of breaking change for finding things in the log?)